lib/ibandit/swedish_details_converter.rb

```diff
@@ -1,5 +1,15 @@
 module Ibandit
   class SwedishDetailsConverter
+    # Converts local Swedish details into SWIFT details.
+    #
+    # Local details can be provided as either:
+    # - branch_code: clearing number, account_number: serial number
+    # - banch_code: nil, account_number: #{clearing number}#{serial number}
+    #
+    # The reverse conversion (extracting local details from SWIFT details) is
+    # not possible, since the clearing number cannot be derived. You should NOT
+    # pass this class a SWIFT account number, as it will not convert it to
+    # local details successfully.
```

Comment:
We can't detect - SWIFT details would look like just passing us an account number, but that's a legitimate argument to pass anyway. Best thing I could think of was this big warning.

In practice I don't think this should be a problem, given the checking we do at the call site (above).